tests/integration/suite/actors/state/http.go

```diff
@@ -77,8 +78,9 @@ func (h *http) Run(t *testing.T, ctx context.Context) {
 	require.NoError(t, err)
 	require.NoError(t, resp.Body.Close())
 	assert.Equal(t, `{"errorCode":"ERR_ACTOR_INSTANCE_MISSING","message":"actor instance is missing"}`, string(b))
-
-	h.app.Daprd().Metrics(t, ctx).MatchMetricAndSum(2, "dapr_error_code_total", "category:actor", "error_code:ERR_ACTOR_INSTANCE_MISSING")
+	assert.Eventually(t, func() bool {
+		return h.app.Daprd().Metrics(t, ctx).MatchMetricAndSum(1, "dapr_error_code_total", "category:actor", "error_code:ERR_ACTOR_INSTANCE_MISSING")
```

Comment:
The value change from 2 to 1 is intentional.
This line wasn't actually evaluated before, that's why we never caught the error.